src/Utility/PowerShellWorkerConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Utility
+{
+    using System;
+
+    internal static class PowerShellWorkerConfiguration
+    {
+        public static string GetString(string name)
```

Comment:
Since these are app settings, would it make sense to add validation to the ```string name```? This way we can detect invalid customer inputs.

Reply:
The `name` value here always comes from our code, not from the customer. The `PowerShellWorkerConfiguration` class implementation always retrieves them from environment variables, so it is not even aware of the concept of app settings.

We may want to perform this validation in the code that copies app settings to environment variables, though. However, this validation is not free, so let's do it when and if we see the need.

Reply:
Sounds good. Thanks for clarifying.